src/embed.js

```diff
@@ -81,9 +81,8 @@ function embed(el, spec, opt) {
     }
     mode = MODES[parsed.library];
 
-    parsedVersion = parsed.version.replace(/^v/g,'');
-    if (versionCompare(parsedVersion, VERSION[mode]) !== 0 ){
-      console.warn("The input spec uses \"" + mode + "\" " + parsedVersion + ", "
+    if (versionCompare(parsed.version, VERSION[mode]) !== 0 ){
```

Comment:
I guess so - but it still shows a warning because pre-release versions are tricky (see the vega/vega#876 ).  BTW, we shouldn't use console.warn - we have log.warn for this.